bin/compile

```diff
@@ -74,11 +74,14 @@ case "$stack" in
       libatk1.0-0
       libatk-bridge2.0-0
       libcairo-gobject2
+      libdrm-dev
+      libgbm-dev
       libgconf-2-4
       libgtk-3-0
       libnspr4
       libnss3
       libx11-xcb1
+      libxcb-dri3-0
```

Comment:
Hi! Thank you for the PR.

To confirm which packages are missing, I created a test repo that contained the following `app.json`:

```json
{
  "stack": "heroku-18",
  "environments": {
    "test": {
      "buildpacks": [
        {"url": "https://github.com/heroku/heroku-buildpack-google-chrome"}
      ],
      "scripts": {
        "test": "ldd $GOOGLE_CHROME_BIN | grep not"
      }
    }
  }
}
```

...and then enabled Heroku CI for it. (I picked Heroku-18 since it had the smallest package list, so will potentially have the most missing)

The output was:

```
	libxcb-dri3.so.0 => not found
	libdrm.so.2 => not found
	libgbm.so.1 => not found
```

As such I think the change needs to be:
```suggestion
      libdrm
      libgbm
      libgconf-2-4
      libgtk-3-0
      libnspr4
      libnss3
      libx11-xcb1
      libxcb-dri3
```

Reply:
I'm more in favour of specifying the version as it will work more consistently on different Debian releases (and fail if the correct version is not available).  I do agree that the `-dev` packages (and thus debugging symbols) are not necessary though.
```suggestion
      libdrm2
      libgbm1
      libgconf-2-4
      libgtk-3-0
      libnspr4
      libnss3
      libx11-xcb1
      libxcb-dri3-0
```

Reply:
Ah you're right the actual package name for `libxcb-dri3` includes the version suffix:
https://packages.ubuntu.com/bionic/libxcb-dri3-0

Reply:
(And same for:
https://packages.ubuntu.com/bionic/libdrm2
https://packages.ubuntu.com/bionic/libgbm1)

Reply:
Same on Debian too and other debian-like distros.  The versioning is part of the package name, it improves reliability of dependancy compatibility.  If it doesn't have the name `libdrm.so.2` on the filesystem it won't be linked by `/lib/ld-linux.so.2` (etc.)

Reply:
Thanks @edmorley, I'm currently @ work, I'll updating everything at noon (in 1 hour) ! I'm already watching the name in Ubuntu 18 of the packages. :)